tests/TestClassDatabaseChanges.php

```diff
@@ -138,9 +138,9 @@ public function testInstallMigrationsDataSimilarEstates(): array
 		$this->assertEquals('Field 1', $newFields[0]);
 		$this->assertEquals('Field 2', $newFields[1]);
 		$this->assertEquals('Field 3', $newFields[2]);
-		$this->assertTrue(true, $newRadius);
```

Comment:
is `assertTrue` correct here? Looks like it should be `assertEquals`?

Reply:
ok let me check it tomorrow